Description: Disable dumping binary strings to stderr while executing tests.
 Modified a build flag controlling verbose tests output that can damage a console
 output.
Forwarded: not-needed
Author: Eugene Seliverstov <theirix@gmail.com>
Last-Update: 2012-09-04
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/asn1c/tests/check-assembly.sh
+++ b/asn1c/tests/check-assembly.sh
@@ -39,7 +39,7 @@
 cat > Makefile <<EOM
 # This file is autogenerated by ../$0
 
-COMMON_FLAGS= -I. -DEMIT_ASN_DEBUG
+COMMON_FLAGS= -I. -DEMIT_ASN_DEBUG=0
 CFLAGS = \${COMMON_FLAGS} ${CFLAGS} -g -O0
 CXXFLAGS = \${COMMON_FLAGS} ${CXXFLAGS}
 
--- a/skeletons/tests/check-OCTET_STRING.c
+++ b/skeletons/tests/check-OCTET_STRING.c
@@ -3,7 +3,7 @@
 #include <time.h>
 #include <sys/time.h>
 
-#define	EMIT_ASN_DEBUG	1
+#define	EMIT_ASN_DEBUG	0
 #include <OCTET_STRING.h>
 #include <BIT_STRING.h>
 
--- a/skeletons/tests/check-REAL.c
+++ b/skeletons/tests/check-REAL.c
@@ -2,7 +2,7 @@
 #include <assert.h>
 #include <math.h>
 
-#define	EMIT_ASN_DEBUG	1
+#define	EMIT_ASN_DEBUG	0
 #include <REAL.h>
 
 static char reconstructed[2][512];
